src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -93,13 +105,33 @@ public void initialize(DeltaTargetContext context) throws Exception {
       .build()
       .getService();
 
-    long maximumExistingSequenceNumber =
-      BigQueryUtils.getMaximumExistingSequenceNumber(context.getAllTables(), project, conf.getDatasetName(),
-                                                     bigQuery, encryptionConfig, MAX_TABLES_PER_QUERY);
+    RetryPolicy<Object> retryPolicy = createBaseRetryPolicy()
+            .handleIf(throwable -> {
+              if (throwable instanceof BigQueryException) {
```

Comment:
Also enable retry for IOException

Reply:
since it doesn't directly throw IOException, will check for cause.